posts/static/makepost/posts.js

```diff
@@ -88,21 +88,20 @@ function submitForm() {
 }
 
 function makePost(imageIDs = undefined) {
+    let authheader = Cookies.get('authheader');
     let content;
     let title = document.getElementById('title').value;
     let description = document.getElementById('description').value;
     let contentType = document.getElementById('contentType').value;
     let visibility = document.getElementById('visibility').value;
     let unlisted = document.getElementById('unlisted').checked;
-    let csrf = document.getElementsByName('csrfmiddlewaretoken')[0].value;
     let visibleToArray = [];
-
-    if (visibility === "PRIVATE") {
-        visibleTo = document.getElementsByClassName('appendedUrl');
+    if (visibility === 'PRIVATE') {
+        let visibleToArray = [];
+        let visibleTo = document.getElementsByName('visibleTo[]');
```

Comment:
This wont submit visibleTo urls on submission of post, name visibleTo[] is no longer used.